static/css/error.less

```diff
@@ -0,0 +1,68 @@
+/* ==========================================================================
+   cfgov-refresh
+   Page-specific styles for the error pages.
+   ========================================================================== */
+
+.btn__error-home {
+    .respond-to-max(@tablet-min, {
+        text-align: center;
+    });
+}
+
+.error_action_container {
+    background-color: @white;
+
+    .respond-to-max(1025px, {
+        .stack-col(content-l_col-1-2);   
+    });
+}
+
+.error_copy_container {
+    .grid_column(6);
+
+    .respond-to-max(800px, {
```

Comment:
Should be `.respond-to-max(799px, {` to be consistent with other breakpoints in the file.